smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/TimestampEncodeGenerator.kt

```diff
@@ -1,43 +0,0 @@
-package software.amazon.smithy.swift.codegen.integration.serde
-
-import software.amazon.smithy.model.traits.TimestampFormatTrait
-import software.amazon.smithy.model.traits.TimestampFormatTrait.Format
-import software.amazon.smithy.swift.codegen.SwiftWriter
-
-class TimestampEncodeGenerator(
```

Comment:
These are deleted since member shape encode takes care of all of this, and is also used where timestamps are decoded outside of members (i.e. for timestamps in headers.)